src/modules/blocks/ticket/container-header/price/style.pcss

```diff
@@ -1,14 +1,15 @@
 .tribe-editor__ticket__container-header-price {
+	color: var(--tec-color-text-primary);
 	flex: none;
-	min-width: 65px;
-	color: #141827;
+	font-family: var(--tec-font-family-sans-serif);
+	font-size: var(--tec-font-size-3);
+	font-style: normal;
+	font-weight: var(--tec-font-weight-bold);
+	line-height: 26px;
```

Comment:
We should try and avoid using pixel line heights.

Typography is set here: https://github.com/the-events-calendar/tribe-common-styles/blob/b3221423195322db1fe9382526c59ce2e462b1ab/variables/_typography.pcss#L37

And here's a reference sheet I created: https://docs.google.com/spreadsheets/d/1Pt2cWr32DzbSeB9AjttQjw_M621gT5NW28xcRBqRDk4/edit?usp=sharing

SO, in this case, with a font size of 16px (via `--tec-font-size-3` above) we'd use `--tec-line-height-3` which would give us a line height of `25.92`

The desire is to stop adding new colors, font sizes, line heights, etc for a more uniform look.

Reply:
Got it. Yeah I saw that 25.92 but I wanted to match figma, so for next time I'll use the closest line-height. Thanks!

Reply:
Suggestion applied, thanks for the explanation